locales/en-US.json

```diff
@@ -469,7 +468,11 @@
       "password_placeholder": "Enter your password",
       "confirm_password_label": "@:common.labels.confirm_password",
       "confirm_password_placeholder": "Confirm your password",
-      "register_button": "Register"
+      "register_button": "Register",
+      "errors": {
+        "PasswordTooShort": "Passwords must be at least 8 characters.",
```

Comment:
Why this limit is hardcoded? Do we have a limit of 8 every time?

Reply:
There is no error's parameters from XAPI. We need to improve xapi.